tools/lint-all

```diff
@@ -83,9 +83,9 @@ def check_pep8(files):
     failed = False
     ignored_rules = [
         'E402', 'E501', 'W503', 'E711', 'E201', 'E203', 'E202', 'E128', 'E226', 'E124', 'E125',
-        'E126', 'E127', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E701', 'E221', 'E303',
+        'E126', 'E231', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E701', 'E221', 'E303',
```

Comment:
This seems a little weird. Why did one of the error codes get moved? You may want to just delete E127 and keep E231 where it was. It will be less likely to make merge conflicts.